Missing docs on imports

With code

```csharp
using System;
using System.IO;
using Logary.Configuration;
using Machine.Specifications;
using NodaTime;
```

There are no docs on what namespaces to import; leads to errors like

    TimeScope_Specs.cs(34,9): error CS0246: The type or namespace name 'It' could not be found (are you missing a using directive or an assembly reference?) [/Users/h/dev/haf/logary/src/Logary.CSharp.Tests/Logary.CSharp.Tests.csproj]

The quickstart is missing these import statements.

Also, there is no `examples/` folder.

--- a/README.md
+++ b/README.md
@@ -14,6 +14,8 @@ MSpec is called a "context/specification" test framework because of the "grammar
 You should be able to see the components of the traditional Arrange-Act-Assert model in there. To support readability and remove as much "noise" as possible, MSpec eschews the traditional attribute-on-method model of test construction. It instead uses custom .NET delegates that you assign anonymous methods and asks you to name them following a certain convention.
 
 ```csharp
+using Machine.Specifications;
+
 [Subject("Authentication")]
 class When_authenticating_an_admin_user
 {

